package.json

```diff
@@ -17,6 +17,7 @@
     "build": "./build",
     "watch": "webpack --progress --colors --watch"
   },
+  "lock": false,
```

Comment:
@amit-mb Can you point me to the documentation for this?

Reply:
Sure, https://docs.npmjs.com/misc/config#package-lock

Reply:
Actually referred this, https://github.com/npm/npm/issues/16867#issuecomment-324554854

Reply:
Thats is just a proposal by someone. I don't think npm have any plan for this to implement.